code/modules/mob/living/say.dm

```diff
@@ -93,6 +93,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!message || message == "")
 		return
 
+	var/static/regex/netspeak = regex("\\b(lol|brb|wtf|idk|omg|lmao|btw|ikr|rofl|tbh|nvm|thx|afaik)\\b", "i")

```

Comment:
add "reddit"

Reply:
Don't tempt me.

Reply:
add "owo" and "uwu" as they are one of the major crimes of this kind

Reply:
please don't use code comments unless you are pointing out an actual code issue or are a maintainer please.

They're otherwise wildly disruptive to the rest of the conversation.